readme.md

```diff
@@ -437,6 +449,78 @@ Scrivito.provideEditingConfig("FormSliderWidget", {
 
 Please note that this example is intended to provide a basic demonstration of creating a custom form widget. It does not include styles or detailed CSS. You can further enhance the appearance and functionality of your custom widgets to match your specific design and usability requirements.
 
+# Captcha Support
```

Comment:
Very good. Something for future - as this becomes bigger and bigger you could think about having just a list of links to create smth like like a Table of Consent to different md files with sections.

Reply:
Good idea, i'll keep that in mind when we add a third captcha